### PR TITLE
Union and Intersection of sorted arrays

### PR DESCRIPTION
**Pull request by Swetha126**
I have contributed the code for union and intersection of two sorted arrays under topic wise questions folder.
I have included clear comments, time and space complexity and sample test case.
Do have a look at the code and let me know the changes if any.
i would like to resolve the issue #82 
Thank you!

### CODE REVIEW

File: Topic Wise Questions with Solutions/Array/intersection_sortedarray.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Enter the elements of array 1 :
+1
+2
+Enter the number of elements of array 2 : 3
+Enter the elements of array 2 :
+1
+3
+4
+Intersection of the two arrays is :
+1
```
Comment (rohanbabbar04) [inline Topic Wise Questions with Solutions/Array/intersection_sortedarray.py:17]: Add the sample output and complexity after the code

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]
Does not give an output when input is
3
8
0
9
2
0
1

File: Topic Wise Questions with Solutions/Array/intersection_sortedarray.py (hunk 1 above)
Comment (Swetha126) [reply Topic Wise Questions with Solutions/Array/intersection_sortedarray.py]: I made the necessary changes by adding the complexity and sample output at the end.

**Comment by Swetha126** [comment]
I looked into the output issue of
3
8 0 9
2
0 1

This problem is caused because in the input, the array is not sorted. Since i was assigned the issue of union and intersection of sorted arrays, i implemented the same.
for the input,
3
0 8 9 
2
0 1
the program works fine.

Thank you!

**Pull request merged**